addon/http-errors/route.js

```diff
@@ -0,0 +1,59 @@
+import Route from '@ember/routing/route';
+import { getOwner } from '@ember/application';
+
+const LIMIT_EXCEEDED = 'LimitExceeded';
+const NOT_FOUND = 'NotFoundError';
+const RESOURCE_NOT_FOUND = 'Resource was not found.';
+const SERVER_ERROR = 'ServerError';
+
+export default class HttpErrorsRoute extends Route {
+  errorValue = null;
+
+  setupController(controller, error) {
```

Comment:
`setupController` didn't have as argument a `controller` and a `model`  ?

Reply:
@tgallice when Ember handles an error, the model is the `error`.

Reply:
Yes, I can change `error` to `model` to avoid confusion

Reply:
OK It's a bit misleading but :D